content/docs/command-reference/add.md

```diff
@@ -93,11 +93,11 @@ reproducible.
   among the `targets`, this option is ignored. For each file found, a new
   DVC-file is created using the process described in this command's description.
 
-- `--no-commit` - do not save outputs to cache. A DVC-file is created, and an
-  entry is added to `.dvc/state`, while nothing is added to the cache. (The
-  `dvc status` command will report that the file is `not in cache`.) This is
-  analogous to using `git add` before `git commit`. Use `dvc commit` when ready
-  to commit the results to cache.
+- `--no-commit` - do not save outputs to cache. A DVC-file is created and an
+  entry is added to `.dvc/state`, while nothing is added to the cache.
+  (`dvc status` will report that the file is `not in cache`.) Use `dvc commit`
+  when ready to commit the results to cache. This is analogous to using
```

Comment:
don't see any improvements to be honest ... commit smth to cache -not sure this is a good idea .. also it's like explaining `commit is commit`

Reply:
There's no change, I only reordered the sentences for consistency with the desc. of this same option in run and repro refs.

Reply:
yep, thus PR is approved :)

Reply:
> commit smth to cache -not sure this is a good idea

Changing to "when ready to commit outputs with DVC" in all refs.

> it's like explaining commit is commit

No because this is the add ref, not the commit ref. It's just a link to the commit ref 🙂